include/dicom/dicom.h

```diff
@@ -1443,11 +1457,11 @@ DcmFilehandle *dcm_filehandle_create_from_memory(DcmError **error,
  * :param error: Error structure pointer
  * :param filehandle: File
  *
- * :return: File Metainformation
+ * :return: File Metadata
  */
 DCM_EXTERN
-DcmDataSet *dcm_filehandle_read_filehandle_meta(DcmError **error, 
-                                                DcmFilehandle *filehandle);
+DcmDataSet *dcm_filehandle_read_file_metadata(DcmError **error, 
```

Comment:
This is not the metadata of the data set (i.e., the collection of non-bulkdata elements), but the [File Meta Information](https://dicom.nema.org/medical/dicom/current/output/chtml/part10/chapter_7.html#sect_7.1). Please change back to "meta". That name was chosen on purpose.

Reply:
OK, I'll change it to `dcm_filehandle_read_file_meta()`